src/backend/distributed/Makefile

```diff
@@ -29,7 +29,9 @@ $(EXTENSION)--5.0.sql: $(EXTENSION).sql
 	cat $^ > $@
 $(EXTENSION)--5.0-1.sql: $(EXTENSION)--5.0.sql $(EXTENSION)--5.0--5.0-1.sql
 	cat $^ > $@
-$(EXTENSION)--5.0-2.sql: $(EXTENSION)--5.0.sql $(EXTENSION)--5.0--5.0-1.sql $(EXTENSION)--5.0-1--5.0-2.sql
+$(EXTENSION)--5.0-2.sql: $(EXTENSION)--5.0-1.sql $(EXTENSION)--5.0-1--5.0-2.sql
+	cat $^ > $@
+$(EXTENSION)--5.1-1.sql: $(EXTENSION)--5.0-2.sql $(EXTENSION)--5.0-2--5.1-1.sql
```

Comment:
So this generates _every_ intermediate version, right? I'd be OK with just generating the latest schema version for each release, but this ladder pattern has a nice regularity.